templates/header.tpl

```diff
@@ -39,7 +39,7 @@
 
 <body class="{bodyClass} skin-{config.bootswatchSkin}">
 	<nav id="menu" class="hidden">
-		<section class="menu-profile">
```

Comment:
Out of curiosity, what the problem with `section` tag here?

Reply:
A section element needs a h1-h6 tag as child:
https://www.w3.org/TR/html5/sections.html#the-section-element

Reply:
Understood, so semantically it does not belong, okay :+1: